### DISCUSSION TITLE
关于项目名称的提问

### DISCUSSION THREAD

**Discussion by xbctmri** [enfein/mieru#182] [Q&A] [answered]
您好！非常感谢您开发了这个出色的项目，项目的名称“見える”让我印象深刻，请问为什么您选择这个名字？它是否有特别的寓意或故事？期待您的解答，再次感谢！

Hello! Thank you for creating this excellent project; The name Mieru left a strong impression on me. May I ask why you chose this name? Does it have a specific meaning or story behind it? Looking forward to your response, and thank you again!

**Comment by enfein** [comment, marked as answer]
参见 https://github.com/enfein/mieru/blob/main/docs/naming.zh_CN.md

**Reply by xbctmri** [reply]
好的，我看明白了，非常感谢。
